src/_scss/pages/homepage/hero/_covid-hero.scss

```diff
@@ -129,13 +156,16 @@ $dot-color: #dcefef;
                     }
                 }
                 p {
-                    color: white;
+                    color: rgb(48, 48, 48);
                     font-size: rem(14);
                     font-weight: $font-normal;
-                    margin-top: rem(10);
-                    @media(min-width: $large-screen) {
+                    margin-top: rem(10);      
+                    @media(min-width: 1300px) {
```

Comment:
why is this different from our standard 1200 ($large-screen)?

Reply:
This is to fix a specific edge case where its breaking at 1300 but not 1200